framework/src/executioners/Eigenvalue.C

```diff
@@ -49,6 +54,10 @@ Eigenvalue::Eigenvalue(const InputParameters & parameters)
   Moose::SlepcSupport::storeSlepcEigenProblemOptions(_eigen_problem, parameters);
   _eigen_problem.setEigenproblemType(_eigen_problem.solverParams()._eigen_problem_type);
 #endif
+
+  if (!parameters.isParamValid("normalization") && parameters.isParamSetByUser("normal_factor"))
+    paramError("normal_factor",
+               "Cannot set scaling factor without defining normalization postprocessor.");
```

Comment:
We actually should allow users to set an arbitrary scaling number if they want.

Reply:
The scaling factor doesn't make much sense if it isn't related to some physical quantity provided by the postprocessor. So @YaqiWang and I think that preventing the user from only scaling the solution with this factor is important. That being said, a user could theoretically trick it by just inputting a constant postprocessor.